phpBB/phpbb/mention/helper/mention_helper.php

```diff
@@ -0,0 +1,211 @@
+<?php
+/**
+*
+* This file is part of the phpBB Forum Software package.
+*
+* @copyright (c) phpBB Limited <https://www.phpbb.com>
+* @license GNU General Public License, version 2 (GPL-2.0)
+*
+* For full copyright and license information, please see
+* the docs/CREDITS.txt file.
+*
+*/
+
+namespace phpbb\mention\helper;
+
+use phpbb\db\driver\driver_interface;
+
+class mention_helper
+{
+	/**
+	* @var driver_interface
+	*/
+	private $db;
+
+	/**
+	 * @var \phpbb\notification\manager
+	 */
+	protected $notification_manager;
+
+	/**
+	* @var Notification Details
+	*/
+	private $data;
+
+	/**
+	* User Mention Helper Constructor
+	*
+	* @param $db   					phpbb\db\driver\driver_interface
+	* @param $notification_manager  \phpbb\notification\manager
+	*
+	* @return \phpbb\mention\helper\mention_helper
+	*/
+	public function __construct(driver_interface $db, \phpbb\notification\manager $notification_manager)
```

Comment:
Please use or the full namespaced name, or a use statement, not both. Look at phpBB on how it is done in general.